rendering.php

```diff
@@ -849,6 +849,15 @@ public function renderNoteText($note) {
 
             $text .= ($publication != '') ? ',' : '.';
         }
+        elseif ($data->has('booktitle')) {  // rwduzhao
+            // $authors? $title //booktitle//, volume, $publication?
```

Comment:
Update comments above (lines 828, 834, 840) to indicate booktitle support. Also I'm not sure if volume is supported. According to line 1023 it's not.